src/intl/zh/page-staking.json

```diff
@@ -196,5 +215,6 @@
   "page-staking-meta-title": "以太坊质押",
   "page-staking-withdrawals-important-notices": "重要通知",
   "page-staking-withdrawals-important-notices-desc": "目前还不能提款。请阅读<a href=\"https://blog.ethereum.org/2021/12/01/eth2-merge-and-post-merge-faq/\" target=\"_blank\">以太坊 2 合并和合并后常见问题</a>，了解更多信息。",
```

Comment:
the url `https://blog.ethereum.org/2021/12/01/eth2-merge-and-post-merge-faq` doesn't exist anymore (we have no post with that slug, https://github.com/ethereum/blog/tree/master/src/posts/en, maybe it was renamed?)

Reply:
This isn't used anywhere.

I've created an issue to get this cleaned up in the repo https://github.com/ethereum/ethereum-org-website/issues/10003.

I think we can just leave this as is since it won't be used anywhere. I can wait until all the April imports and resolve this in the linked issue.